clang/test/SemaSYCL/union-kernel-param1.cpp

```diff
@@ -0,0 +1,63 @@
+// RUN: %clang_cc1 -I %S/Inputs -fsycl -fsycl-is-device -ast-dump %s | FileCheck %s
+
+// This test checks that compiler generates correct kernel arguments for
+// unions containing Arrays.
+
+#include <sycl.hpp>
+
+using namespace cl::sycl;
```

Comment:
I don't see usage of this include. Let's remove it.

Reply:
Done